Spider (mob)/spider.gd

```diff
@@ -0,0 +1,8 @@
+extends CharacterBody2D
+
+@export var speed = 200
+
+func _physics_process(delta):
+	var direction = global_position.direction_to(Global.player.global_position)
+	velocity = direction * 300.0
```

Comment:
Looks like you forgot to use the speed variable here.

Reply:
yes, i fixed it